### DISCUSSION TITLE
Elenco fatture di un cliente

### DISCUSSION THREAD

**Discussion by creativalab** [fattureincloud/api#405] [Q&A] [answered]
Salve, qual è la chiamata per ottenere il totale delle fatture di uno specifico cliente?

**Comment by creativalab** [comment]
Se faccio una chiamata tipo /c/XXX/issued_documents?type=invoice&per_page=5&sort=-id&page=1&q=entity.vat_number=XXXXX questa è la risposta:

cURL Error #:Operation timed out after 30002 milliseconds with 0 bytes received

**Comment by emajo** [comment, marked as answer]
Ciao @creativalab, il totale delle fatture emesse verso un cliente non è un informazione che stiamo fornendo tramite api, la chiamata di list issued documents serve per ottenere i documenti completi e non risulta essere filtrabile per cliente.

Riguardo invece l'errore curl è probabilmente dovuto a un errata configurazione della chiamata.

Grazie,
Emanuele.
